mapdata/WarcraftLegacies/Units/9728_8704.json

```diff
@@ -10,21 +10,21 @@
     "AbilityData": [],
```

Comment:
This file has a conflict because you changed this area in #2765 then are also changing it separately here.
So to fix that I suggest:

1. Pull from `dev` into this branch
2. When it prompts you, overwrite this file with the one from `dev`
3. Redo the change you want in this area (I imagine you removed a Shredder from the map or something)

Reply:
I went ahead and did this for you. It's just easier because I caused a tonne of other merge conflicts with some system changes.